lib/svg_sigs.mli

```diff
@@ -673,7 +673,7 @@ module type T = sig
   val a_onunload : Xml.event_handler  -> [> | `OnUnload ] attrib
   val a_onzoom : Xml.event_handler  -> [> | `OnZoom ] attrib
 
-  (** {3 Javascript mouse events} *)
+  (** {6 Javascript mouse events} *)
```

Comment:
Why changing the header level ?

Reply:
Oh lol – I obviously didn't get what that was at all.